app/templates/components/user-cards.hbs

```diff
@@ -11,17 +10,46 @@
             {{#each teams as |team|}}
                {{#ddm.item}}
                   <a href {{ action "getTeams" team }}>
-                     <div class ="item-lists">{{team}}</div>
+                     <div class ="item-lists item-droplists">{{team}}</div>
                   </a>
                {{/ddm.item}}
             {{/each}}
          {{/dd.menu}}
       </BsDropdown>
    </div>
+
+   <div class = "dropdown-container">
+      <div class = "dropdown-text">Sort By</div>
+         <BsDropdown @closeOnMenuClick={{false}} as |dd|>    
+            <dd.toggle><div class ="option-text">Options </div>
+               {{svg-jar 'Vector' class='vector' width='8px' height='5px'}}
+            </dd.toggle>
+            {{#dd.menu as |ddm| }}
+               {{#each sortfieldList as |fieldList|}}
+                  {{#ddm.item}}
+                     <a href {{ action "sortByFields" fieldList.field }}>
+                     <div class ="item-lists {{if (eq sort_option fieldList.field) "selected"  }}">{{fieldList.title}}</div>
+                     </a>
+                  {{/ddm.item}}
+               {{/each}}
+               <div class="dropdown-divider"></div>
+               {{#each sortorderlist as |orderList|}}
+                  {{#ddm.item}}
+                     <a href {{ action "sortByFieldsInOrder" orderList.title }}>
```

Comment:
The reason we had object like structure in `fieldList` is to display `title` and call actions with `field` like how we have done with `sortByFields`. This way it can be clean.

Reply:
Done.